extensions/ql-vscode/src/view/method-modeling/MethodModelingInputs.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { styled } from "styled-components";
+import { Method } from "../../model-editor/method";
+import { ModeledMethod } from "../../model-editor/modeled-method";
+import { ModelTypeDropdown } from "../model-editor/ModelTypeDropdown";
+import { ModelInputDropdown } from "../model-editor/ModelInputDropdown";
+import { ModelOutputDropdown } from "../model-editor/ModelOutputDropdown";
+import { ModelKindDropdown } from "../model-editor/ModelKindDropdown";
+
+const Container = styled.div`
+  padding-top: 0.5rem;
+`;
+
+const Label = styled.span`
+  display: block;
+  padding-bottom: 0.3rem;
+`;
+
+export type MethodModelingInputsProps = {
+  method: Method;
+  modeledMethod: ModeledMethod | undefined;
+  onChange: (method: Method, modeledMethod: ModeledMethod) => void;
+};
+
+export const MethodModelingInputs = ({
+  method,
+  modeledMethod,
+  onChange,
+}: MethodModelingInputsProps): JSX.Element => {
+  const inputProps = {
+    method,
+    modeledMethod,
+    onChange,
+  };
+
+  return (
+    <>
+      <Container>
+        <Label>Model Type</Label>
```

Comment:
Could this label be an actual [`<label>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) element with the correct accessibility attributes (either the `<select>` is a child of the `<label>` or the `htmlFor` attribute is set)? Then we also don't need the `aria-label` on the `<select>` itself anymore for these elements.

Reply:
Ah good point, thanks for thinking about accessibility :) I've left the aria-label stuff because the components are also used in the main editor grid.